rpcs3/Emu/RSX/Overlays/overlay_animation.cpp

```diff
@@ -0,0 +1,74 @@
+#include "stdafx.h"
+#include "overlay_animation.h"
+#include "overlay_controls.h"
+
+namespace rsx
+{
+    namespace overlays
+    {
+        void animation_translate::apply(compiled_resource& resource)
+        {
+            if (!progress.x && !progress.y && !progress.z) return;
```

Comment:
You kinda break your own style here by not using a scoped return, as seen below.